cypress/integration/Services/Measure Service/Measure.spec.ts

```diff
@@ -40,6 +40,46 @@ describe('Measure Service: Create Measure', () => {
         })
     })
 
+    //Get All Measures
+    it('Get all Measures', () => {
+
+        cy.getCookie('accessToken').then((accessToken) => {
+            cy.request({
+                url: '/api/measures',
+                method: 'GET',
+                headers: {
+                    authorization: 'Bearer ' + accessToken.value
+                }
+            }).then((response) => {
+                expect(response.status).to.eql(200)
+                expect(response.body).to.not.be.null
+                expect(response.body).to.be.a('array')
+                cy.get(response.body.length)
+                expect(response.body[0].id).to.be.exist
+            })
+        })
+    })
+
+    //Get Measures by User
+    it('Get all Measures created by logged in User', () => {
+
+        cy.getCookie('accessToken').then((accessToken) => {
+            cy.request({
+                url: '/api/measures?currentUser=true',
+                method: 'GET',
+                headers: {
+                    authorization: 'Bearer ' + accessToken.value
+                }
+            }).then((response) => {
+                expect(response.status).to.eql(200)
```

Comment:
are we also checking the array length verifying that?

Reply:
Yes this line ' cy.get(response.body.length)' will give the length of the array